src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -155,6 +155,10 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
                       + "is not associated with an Institution.");
                 }
             }
+            if (roleName.equalsIgnoreCase(UserRoles.RESEARCHER.getRoleName())) {
+                    List<DataAccessRequest> dars = dataAccessRequestDAO.findAllDarsByUserId(user.getDacUserId());
+                    return createAccessRequestManageV2(dars);
+            }
         }
         //if there is no roleName then user is a member, chair, or admin
         List<DataAccessRequest> allDars = findAllDataAccessRequests();
```

Comment:
So, this method actually DOES support member/chair/admin but the null case has semantic meaning. A better approach to this method might be to handle the member/chair/admin cases all in the same switch block since they are processed using the same code.